src/normalize.jl

```diff
@@ -0,0 +1,77 @@
+using LinearAlgebra
+
+function rescale(tn::AbstractITensorNetwork; alg="exact", kwargs...)
```

Comment:
Is the rescaling being done here a normalization? If so I would call this `normalize`.

Reply:
Ignore this, I now remember that we deliberately had functions `rescale` and `normalize` to distinguish between closed networks and states.